src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] arrayOfFigure = new Figure[6];
+        final int First_Half_Of_Array = 3;
+
+        for (int i = 0; i < First_Half_Of_Array; i++) {
+            arrayOfFigure[i] = FigureSupplier.getRandomFigure();
+        }
+        for (int i = First_Half_Of_Array; i < arrayOfFigure.length; i++) {
+            arrayOfFigure[i] = FigureSupplier.getDefaultFigure();
+        }
+        for (int i = 0; i < arrayOfFigure.length; i++) {
+            arrayOfFigure[i].draw();
+            if (i >= First_Half_Of_Array || i == 1) {
```

Comment:
The condition `i == 1` seems to be incorrect because it will only add a newline after the second figure. If you want to add a newline after each figure is drawn, you can remove the condition and always print the newline, or adjust the condition to check if `i` is less than `arrayOfFigure.length - 1` to avoid an extra newline at the end.